### DISCUSSION TITLE
code to send an email

### DISCUSSION THREAD

**Discussion by dscarlos72** [Budibase/budibase#10307] [Help] [answered]
hello,
It is possible to insert a javascript code to send an email 30 days before a defined date ?

thanks in advance

**Comment by melohagan** [comment, marked as answer]
Hey @dscarlos72 

Yes this would be possible with a flow something like this:

1. **Cron trigger** - run once every day
2. **Condition** - Is today's date 30 days before a given date (if this date needs to come from somewhere then you could do a Query Rows before the condition step)
3. **Send Email**

This page in the docs should give you ideas! https://docs.budibase.com/docs/conditions-1
